components/TProfileDetails.vue

```diff
@@ -1,46 +1,46 @@
 <template>
   <div class="space-y-4 mt-8 p-4 bg-gray-100">
     <dl v-if="profile.locales">
-      <dt class="font-bold mr-1">Languages:</dt>
+      <dt class="font-bold mr-1">{{ $t('TProfileDetailes.languages') }}:</dt>
```

Comment:
Those are just fields from `use/profile.js`, so maybe reuse existing objects to keep translations in one group?